### PR TITLE
Deep sleep mode

### PR DESCRIPTION
**Pull request by gemenerik**
Allow deep sleep to be triggered from the base station

### CODE REVIEW

File: src/main/drivers/sound_beeper.c [hunk 1]
```diff
@@ -123,7 +123,7 @@ void beeperInit(const beeperDevConfig_t *config)
             IOInit(beeperIO, OWNER_BEEPER, 0);
             IOConfigGPIO(beeperIO, config->isOpenDrain ? IOCFG_OUT_OD : IOCFG_OUT_PP);
         }
-        systemBeep(false);
+        systemBeep(true);
```
Comment (kevindehecker) [inline src/main/drivers/sound_beeper.c]: Is this necessay?

File: .vscode/settings.json [hunk 2]
```diff
@@ -6,6 +6,6 @@
         "unordered_set": "c",
         "regex": "c",
         "io.h": "c",
-        "stdbool.h": "c"
+        "stdbool.h": "c",
```
Comment (kevindehecker) [inline .vscode/settings.json]: Remove unnecessary change

File: src/main/fc/core.c [hunk 3]
```diff
@@ -757,12 +757,24 @@ bool isAirmodeActivated()
     return airmodeIsActivated;
 }
 
-
```
Comment (kevindehecker) [inline src/main/fc/core.c]: Remove unnecessary change.

File: src/main/drivers/sound_beeper.c (hunk 1 above)
Comment (kevindehecker) [reply src/main/drivers/sound_beeper.c]: Ah, I see now that's how we hacked it :)

Can this be done from the BF settings instead?

Comment (kevindehecker) [reply src/main/drivers/sound_beeper.c]: We need to keep BF changes to a minimum, because it is a merge hell otherwise when we upgrade to a newer BF.

File: src/main/fc/core.c [hunk 4]
```diff
@@ -757,12 +757,24 @@ bool isAirmodeActivated()
...
 {
+    static bool hard_shutdown = false;
+
+    if(rcData[AUX2] > PATS_SLEEP_MIN && rcData[AUX2] < PATS_SLEEP_MAX) {
+        hard_shutdown = true;
+    }
+
+    if (hard_shutdown) {
+        systemBeep(false);
+    }
```
Comment (kevindehecker) [inline src/main/fc/core.c]: } else {

**Review by kevindehecker — CHANGES_REQUESTED** [review]

File: src/main/drivers/sound_beeper.c (hunk 1 above)
Comment (gemenerik) [reply src/main/drivers/sound_beeper.c]: Maybe @woutervdhof has some intuition on whether or not this should be possible to do from BF settings?

Comment (gemenerik) [reply src/main/drivers/sound_beeper.c]: Found a configuration option to invert the systemBeep, testing now

Comment (gemenerik) [reply src/main/drivers/sound_beeper.c]: Success

Comment (gemenerik) [reply src/main/drivers/sound_beeper.c]: * not a success. Inverting triggers sleep at low voltage immediately. Probably fixable but would require even more changes = merge hell
